src/utils/ng-validate.ts

```diff
@@ -4,13 +4,15 @@ import {MessageType} from '../communication/message-type';
 import {send} from '../backend/indirect-connection';
 
 declare const getAllAngularTestabilities: Function;
+declare const getAllAngularRootElements: Function;
+declare const ng: any;
 
 let unsubscribe: () => void;
 
 const handler = () => {
   // variable getAllAngularTestabilities will be defined by Angular
```

Comment:
@igor-ka Previous version behaviour should stay the same, but this comment should be updated, and your comment prompted me to think about some other things. Let me improve this PR a little.